pkg/internal/checkup/executor/executor.go

```diff
@@ -257,34 +261,70 @@ func extractSectionStatistics(input, sectionStart, sectionEnd string) (string, e
 	return strings.Join(lines[startLineIdx+1:endLineIdx], "\n"), nil
 }
 
-func parseTestpmdStats(input string) (map[string]int64, error) {
-	params := make(map[string]int64)
-	scanner := bufio.NewScanner(strings.NewReader(input))
-	for scanner.Scan() {
-		line := scanner.Text()
-		if line == "" {
-			continue
+func parseTestpmdStats(input string) ([]testPmdPortStats, error) {
+	statistics := make([]testPmdPortStats, testPmdPortStatsSize)
+	const (
+		port0SectionStart   = "Forward statistics for port 0"
+		port0SectionEnd     = "----------------------------------------------------------------------------"
+		port1SectionStart   = "Forward statistics for port 1"
+		port1SectionEnd     = port0SectionEnd
+		SummarySectionStart = "Accumulated forward statistics for all ports+"
+		SummarySectionEnd   = "++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++"
+	)
+
+	statsIdxToStartSectionMap := map[testPmdStatsIdx]string{
+		testPmdStatsPort0:       port0SectionStart,
+		testPmdStatsPort1:       port1SectionStart,
+		testPmdPortStatsSummary: SummarySectionStart,
+	}
+	statsIdxToEndtSectionMap := map[testPmdStatsIdx]string{
+		testPmdStatsPort0:       port0SectionEnd,
+		testPmdStatsPort1:       port1SectionEnd,
+		testPmdPortStatsSummary: SummarySectionEnd,
+	}
+
+	for statsIdx := testPmdStatsPort0; statsIdx < testPmdPortStatsSize; statsIdx++ {
```

Comment:
I find this whole thing over complected.
How about:
```Go
startSections := [testPmdPortStatsSize]string{port0SectionStart, port1SectionStart, SummarySectionStart}
endSections := [testPmdPortStatsSize]string{port0SectionEnd, port1SectionEnd, SummarySectionEnd}
for statsIdx := range startSections {
    sectionString, err := extractSectionStatistics(input, startSections[statsIdx], endSections[statsIdx])
    ...
}
```

Reply:
DONE :)